mmcv/runner/dist_utils.py

```diff
@@ -47,9 +50,17 @@ def init_dist(launcher, backend='nccl', **kwargs):
 def _init_dist_pytorch(backend, **kwargs):
     # TODO: use local_rank instead of rank % num_gpus
     rank = int(os.environ['RANK'])
-    num_gpus = torch.cuda.device_count()
-    torch.cuda.set_device(rank % num_gpus)
-    dist.init_process_group(backend=backend, **kwargs)
+    if hasattr(torch, 'is_mlu_available') and torch.is_mlu_available():
```

Comment:
Does MLU support other launchers like `MPI` or `Slurm`?

Reply:
Nope, MLU only supports PyTorch backend and PyTorch launcher.